agir/front/components/formComponents/CommentField.js

```diff
@@ -281,8 +281,12 @@ const CommentField = (props) => {
   const maySend = !isLoading && value && value.trim().length <= 1000;
 
   const updateScroll = () => {
-    setTimeout(() => {
-      scrollerRef.current.scrollTo(0, scrollerRef.current.scrollHeight);
+    const pid = setInterval(() => {
+      const scrollerElement = scrollerRef.current;
+      if (!!scrollerElement) {
+        scrollerElement.scrollTo(0, scrollerElement.scrollHeight);
+        clearInterval(pid);
+      }
```

Comment:
Pourquoi faire un `setInterval` ?

Reply:
Pour réitérer l'opération tant que que l'élément n'est pas présent.
Il me semble que l'erreur vient de là, 50ms est trop peu pour certains device à priori ?

Reply:
Le problème c'est pour le `isFocused` en fait.
Quand on clique sur lle `TextField` pour écrire un message le `isFocused` passe à `True` et est appelé `updateScroll`.
Or on l'appelle justement à ce moment car le `TextField` se déplie mais prend du temps à se déplier.

Tu vois une manière de détecter quand le composant TextField a fini de s'aggrandir ?